src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -157,4 +157,9 @@ internal partial class BoundPassByCopy
     {
         public override object Display => Expression.Display;
     }
+
+    internal partial class UnboundObjectCreationExpression
+    {
+        public override object Display => Syntax;
```

Comment:
>Syntax [](start = 42, length = 6)

This is kind of nasty as it includes unbounded source text. I would suggest something more like `"new (...)"`